src/interfaces/change-firm-interfaces/change-firm-interface.ts

```diff
@@ -0,0 +1,8 @@
+import { FilingBusinessIF, FilingHeaderIF } from '@/interfaces'
+
+/** Interface for data object UI sends to API. */
+export interface ChangeFirmIF {
+  header: FilingHeaderIF
+  business: FilingBusinessIF
+  changeFirm: any
```

Comment:
No type here? Or union type (A | B) ?

Reply:
Not yet, there is a few places that i left the Type undefined as we didn't have a model/schema for Sp/Gps quite yet at the time of writing this, so i opted to leave it until we did know. 

In the process of creating some Sp/Gp records to help model out these entity types that don't/didn't currently exist.